demo/atom/button/demo/index.scss

```diff
@@ -0,0 +1,6 @@
+.AtomButtonLinkEllipsis {
+  width: 60px;
```

Comment:
Why 60px? It should be a variable or maybe prop to be able to adjust it depending on the need?